stl/src/excptptr.cpp

```diff
@@ -37,13 +37,34 @@ extern "C" _CRTIMP2 void* __cdecl __AdjustPointer(void*, const PMD&); // defined
 using namespace std;
 
 namespace {
-#ifdef _M_CEE_PURE
+#if defined(_M_CEE_PURE)
     template <class _Ty>
     _Ty& _Immortalize() { // return a reference to an object that will live forever
         /* MAGIC */ static _Immortalizer_impl<_Ty> _Static;
         return reinterpret_cast<_Ty&>(_Static._Storage);
     }
-#else // ^^^ _M_CEE_PURE ^^^ // vvv !_M_CEE_PURE vvv
+#elif _MSC_VER > 1927 && !defined(_M_CEE) // _M_CEE test is TRANSITION, VSO-1153256
+    template <class _Ty>
+    struct _Constexpr_excptptr_immortalize_impl {
+        union {
+            _Ty _Storage;
+        };
+
+        constexpr _Constexpr_excptptr_immortalize_impl() : _Storage() {}

```

Comment:
```suggestion
        constexpr _Constexpr_excptptr_immortalize_impl() : _Storage() {}
        _Constexpr_excptptr_immortalize_impl(const _Constexpr_excptptr_immortalize_impl&) = delete;
        _Constexpr_excptptr_immortalize_impl& operator=(const _Constexpr_excptptr_immortalize_impl&) = delete;
```

Reply:
Don't forget Stephan's "blank lines between defaulted/deleted and non-trivial functions" rule.

Reply:
If Casey were here, he would say that if Stephan were here, he would say that...